File: Microservices/BillingService/BillingService.Domain/Models/ClaimDiagnosisCodeModel.cs
namespace BillingService.Domain.Models
{
    public class ClaimDiagnosisCodeModel
    {
        public int DiagnosisId { get; set; }
        public string DiagnosisCode { get; set; }
        public string Description { get; set; }
        public int Order { get; set; }
        public bool IncludeOnClaims { get; set; }
    }
}

File: billing_ui_app/src/app/core/models/clients/claim-diagnosis-code-model.ts
export class ClaimDiagnosisCodeModel {
    diagnosisId: number;
    diagnosisCode: string;
    diagnosisDescription: string;
    diagnosisFullDescription: string;
    description: string;
    order: number;
    includeOnClaims: boolean;
    startDate: Date;
    endDate?: Date;
}